.travis.yml

```diff
@@ -1,16 +1,17 @@
+dist: bionic
 language: go
-sudo: required
 
 go:
-  - 1.12.x
+  - 1.13.x
 
-go_import_path: github.com/containerd/release-tool
+env:
+  - GOLANGCI_LINT_VERSION=v1.22.2
 
 install:
   - go get -u github.com/vbatts/git-validation
```

Comment:
you cannot call `go get` in mod repo because `go get` will update go.mod and go.sum files. 

in order to avoid this case, `pushd ..; go get xxx; popd` will help

Reply:
Yeah, I agree, but the updated files never got committed, right?

Reply:
But the `../project/script/validate/vendor` will fail

Reply:
Now I understand how that `validate/vendor` step works. I I will bring it back.

Reply:
I will take it back, looks like the vendor checks runs into `??` folder issue.

```
?? vendor/golang.org/x/sys/README.md
?? vendor/golang.org/x/sys/cpu/
?? vendor/golang.org/x/sys/cpu/cpu.go
?? vendor/golang.org/x/sys/cpu/cpu_arm.go
?? vendor/golang.org/x/sys/cpu/cpu_arm64.go
?? vendor/golang.org/x/sys/cpu/cpu_gc_x86.go
?? vendor/golang.org/x/sys/cpu/cpu_gccgo.c
?? vendor/golang.org/x/sys/cpu/cpu_gccgo.go
?? vendor/golang.org/x/sys/cpu/cpu_mips64x.go
?? vendor/golang.org/x/sys/cpu/cpu_mipsx.go
?? vendor/golang.org/x/sys/cpu/cpu_ppc64x.go
?? vendor/golang.org/x/sys/cpu/cpu_s390x.go
?? vendor/golang.org/x/sys/cpu/cpu_x86.go
?? vendor/golang.org/x/sys/cpu/cpu_x86.s
?? vendor/golang.org/x/sys/windows/
?? vendor/golang.org/x/sys/windows/svc/
?? vendor/golang.org/x/sys/windows/svc/event.go
?? vendor/golang.org/x/sys/windows/svc/go12.c
?? vendor/golang.org/x/sys/windows/svc/go12.go
?? vendor/golang.org/x/sys/windows/svc/go13.go
?? vendor/golang.org/x/sys/windows/svc/security.go
?? vendor/golang.org/x/sys/windows/svc/service.go
?? vendor/golang.org/x/sys/windows/svc/sys_386.s
?? vendor/golang.org/x/sys/windows/svc/sys_amd64.s
The command "../project/script/validate/vendor" exited with 1.
```

Reply:
For the PR's sake, I will just remove the vendor validate step (which is pretty much consistent with what is doing in the other repos)